.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,144 @@
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
+name: Bug Report
+description: Report a bug
+
+body:
+
+  - type: checkboxes
+    attributes:
+      label: Contributing guidelines
+      description: |
+        Please read the contributing guidelines before proceeding.
+      options:
+        - label: I've read the [contributing guidelines](https://github.com/docker/buildx/blob/master/.github/CONTRIBUTING.md) and wholeheartedly agree
+          required: true
+
+  - type: checkboxes
+    attributes:
+      label: I've found a bug and checked that ...
+      description: |
+        Make sure that your request fulfills all of the following requirements.
+        If one requirement cannot be satisfied, explain in detail why.
+      options:
+        - label: ... the documentation does not mention anything about my problem
+        - label: ... there are no open or closed issues that are related to my problem
+
+  - type: textarea
+    attributes:
+      label: Description
+      description: |
+        Please provide a brief description of the bug in 1-2 sentences.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Expected behaviour
+      description: |
+        Please describe precisely what you'd expect to happen.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Actual behaviour
+      description: |
+        Please describe precisely what is actually happening.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: |
+        Please describe the steps to reproduce the bug.
+      placeholder: |
+        1. ...
+        2. ...
+        3. ...
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Buildx version
+      description: |
+        Can be found using the `docker buildx version` command.
+        Example: `github.com/docker/buildx v0.8.1 5fac64c2c49dae1320f2b51f1a899ca451935554`
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: BuildKit version
+      description: |
+        Please provide the BuildKit version being used by your builder.
```

Comment:
We should provide info to how to get this information as well - I know the unreleased `buildx ls` has this, maybe we can call that out here? Or maybe just remove this section, since we ask for that later anyways?

Reply:
I think we can remove this now that buildx 0.9 with `buildx ls` displays BuildKit version.

Reply:
Would it be more useful to have the information of the currently active builder (instead of `buildx ls` with all builders?)

Reply:
`buildx inspect` unfortunately doesn't print the buildkit version! That seems like an oversight, I'll take a look at that. I think atm `buildx ls` is the only way to get that info.

There's also a number of cases where having all the builders listed is super helpful for debugging, especially when doing things around builder management/etc.

Reply:
Yeah `buildx ls` is more useful also `*` and `--builder` indicated which builder is being used from the repro.